org.sugarlabs.AbacusActivity.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "org.sugarlabs.AbacusActivity",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "separate-locales": false,
+    "build-options": {
+        "env": {
+            "PERL5LIB": "/app/lib/perl5/site_perl/5.30.0/",
+            "PYTHON_VERSION": "3"
+        }
+    },
+    "command": "sugarapp",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+        "--env=SUGAR_BUNDLE_ID=org.sugarlabs.AbacusActivity",
+        "--env=SUGAR_BUNDLE_PATH=/app/share/sugar/activities/Abacus.activity",
+        "--env=GDK_BACKEND=x11"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        "sugar-shared-modules/sugar/artwork.json",
+        "sugar-shared-modules/sugar/toolkit-gtk3.json",
+        "sugar-shared-modules/python/sugarapp.json",
+        {
+            "name": "abacus",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python${PYTHON_VERSION} setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
```

Comment:
Any reason for doing this? We don't have python2 anymore in the runtime, you can use python3

Reply:
Not in this specific case (I will remove it) but, for the dependencies, Sugar is currently porting from python2 to python3, so there is a huge mix between 2 and 3 apps. I did this for not having to duplicate all these dependency files.